binr/ragg2/ragg2.c

```diff
@@ -315,9 +315,13 @@ int main(int argc, char **argv) {
 
 	// catch this first
 	if (get_offset) {
-		get_offset = r_num_math (0, sequence);
-		printf ("Little endian: %d\n", r_debruijn_offset (get_offset, false));
-		printf ("Big endian: %d\n", r_debruijn_offset (get_offset, true));
+		if (!strncmp (sequence, "0x", 2)) {
```

Comment:
If there's an error it should return 1.

but actually, it should work with other bases.. but base16 is probably the most expected one and can solve some confussions.. i guess